Sso brute force (#36618)
demisto/content: Packs/Core/ReleaseNotes/3_0_71.md

#### Playbooks

##### New: SSO Brute Force

> - New: This playbook is designed to address the following alerts:

Reviewer: ```suggestion
New: This playbook addresses the following alerts:
```

- SSO Brute Force
- SSO Brute Force Threat Detected
- SSO Brute Force Activity Observed

Playbook Stages:
Triage:

> - The playbook will check the IP reputation and will fetch the events related to the brute force login attempts.
> 
> Early Containment:
> 
> - The playbook will check if the IP is suspicious. If it is, the playbook will suggest blocking the IP using PAN-OS. The investigation will continue in parallel to this phase.
> 
> Investigation:
> 
> - The playbook will assess the risk score of the user who successfully logged in after a brute force attempt, examine the legitimacy of the user agent and if the brute force attempt is likely automated based on the timestamp interval. It will also verify if the user has MFA configured when the alert source is Okta.

Reviewer: ```suggestion
- The playbook checks the IP reputation and fetches the events related to the brute force login attempts.

Early Containment:

- The playbook checks if the IP is suspicious. If it is, the playbook suggests blocking the IP using PAN-OS. The investigation continues in parallel to this phase.

Investigation:

- The playbook assesses the risk score of the user who successfully logged in after a brute force attempt, examines the legitimacy of the user agent and if the brute force attempt is likely automated based on the timestamp interval. It also verifies if the user has MFA configured when the alert source is Okta.
```

Containment:

> - If the user's risk score is high, or if the user agent is detected as suspicious, or if the time intervals indicates that the login attempts is likely automated, and there is a successful login attempt, the playbook will clear the user's session. 
> If the user doesn't have MFA configured, the playbook will recommend expiring the user's password. If there is no successful login detected, the playbook will add the investigation findings as a note.

Reviewer: ```suggestion
- If the user's risk score is high, or if the user agent is detected as suspicious, or if the time intervals indicates that the login attempts is likely automated, and there is a successful login attempt, the playbook clears the user's session. 
If the user doesn't have MFA configured, the playbook recommends expiring the user's password. If there is no successful login detected, the playbook adds the investigation findings as a note.
```

Requirements:

> For any response action, you will need one of the following integrations:
> 
> - Microsoft Graph User
> - Okta v2
> 
> For eradication step,  you will need the following integration:

Reviewer: ```suggestion
For any response action, you need one of the following integrations:

- Microsoft Graph User
- Okta v2

For eradication step,  you need the following integration:
```

- Palo Alto Networks PAN-OS.<~XSIAM> (Available from Cortex XSIAM).</~XSIAM>

#### Triggers Recommendations

##### New: SSO Brute Force

> - New: This trigger is responsible for handling the 'SSO Brute Force Threat Detected' and 'SSO Brute Force Activity Observed' alerts<~XSIAM> (Available for Cortex XSIAM).</~XSIAM>

Reviewer: ```suggestion
New: This trigger is responsible for handling the 'SSO Brute Force Threat Detected' and 'SSO Brute Force Activity Observed' alerts<~XSIAM> (Available for Cortex XSIAM).</~XSIAM>
```
